tutorials/create-serverless-scraping/index.mdx

```diff
@@ -0,0 +1,336 @@
+      }
+      secret_environment_variables = {
+        DB_PASSWORD = scaleway_rdb_user.worker.password
+      }
+    }
+    ```
+7. Provision the triggers resources. The cron trigger activates at the minutes `[0, 15, 30, 45]` of every hour. No argument is passed, but we could do so by specifying them in json format in the `args` parameter. 
+    ```
+    resource "scaleway_function_cron" "scraper_cron" {
+      function_id = scaleway_function.scraper.id 
+      schedule = "0,15,30,45 * * * *"
+      args = jsonencode({})
+    }
+
+    resource "scaleway_function_trigger" "consumer_sqs_trigger" {
+      function_id = scaleway_function.consumer.id 
+      name = "hn-sqs-trigger"
+      sqs {
+        project_id = scaleway_mnq_sqs.main.project_id
+        queue = scaleway_mnq_sqs_queue.main.name
+      }
+    }
+    ```
+
+## Put it all in motion
+Using terraform, it is really straightforward: just run
+```
+cd terraform
+terraform init
+terraform plan 
+terraform apply
+```
+
+How to know if it is working as intended? 
+
+First, you can go in the scaleway console, [activate cockpit](https://www.scaleway.com/en/docs/observability/cockpit/quickstart/), and check logs and metrics about the functions execution or queue stats. 
+
+And to check if the data is correctly stored in the database, you can try to connect directly to it via a cli tool like psql. 
+You can retrieve your instance ip and port from the console, under the managed db section.
+When prompted for password, you can find it by running `terraform output -json`
+```
+psql -h <DB_INSTANCE_IP> --port <DB_INSTANCE_PORT> -d hn-database -U worker
+```
+
+And when you are done testing, don't forget to clean up! First deactivate the cockpit from the console, if you had activated it. Then run: 
+```
+cd terraform
+terraform destroy
+```
+
+## Summary, going further, key takeaways
+We have showed how to asynchronously decouple producer and consumer using SQS, adhering to serverless patterns. 
+While the volume of data processed in this example is not really notewhorty, thanks to queue's robustness and functions 
+auto scaling capabilities, it is possible to manage larger workloads. 
+
+Here are some possible extensions to this basic example:
+ - Replace the simple proposed logic with your own. What about counting how many times some keywords (e.g: copilot, serverless, microservice) appears in HN articles? 
+ - Define multiple cron triggers for different websites and pass the website as argument to the function. Or create multiple functions that feed the same queue. 
+ - Use a serverless container instead of the consumer function, and use a command line util like `htmldoc` or `pandoc` to perform a pdf conversion of the article and upload the result to an S3 bucket. 
+ - Replace the managed rdb with a Scaleway serverless db, so that all the infrastructure lives in the serverless ecosystem! *Note that at the moment there is no terraform support for serverless db, hence the choice to use rdb*.
```

Comment:
```suggestion
 - Replace the Managed Database for PostgreSQL with a [Scaleway Serverless Database](/serverless/sql-databases/quickstart/), so that all the infrastructure lives in the serverless ecosystem! *Note that at the moment there is no Terraform support for Serverless Database, hence the choice here to use Managed Database for PostgreSQL*.
```